python_client/slicer_package_manager_client/__init__.py

```diff
@@ -93,7 +97,7 @@ def createApp(self, name, desc=None, coll_id=None, coll_name=None, coll_desc=Non
         """
         apps = self.listApp(name=name)
         if apps:
-            raise Exception('The Application "%s" already exist.' % name)
+            raise SlicerPackageManagerError('Error: The Application "%s" already exist.' % name)
```

Comment:
You don't have to include the string `Error:`, it is when the exception is caught that the code could decide to use the prefix `Error: `